resources/assets/less/bem/osu-md.less

```diff
@@ -66,6 +66,10 @@
       margin-top: 0;
     }
 
+    & > a {
```

Comment:
`&` optional. And probably should look into setting `&[id], [id] { ... }` instead (not sure if better).